src/App/Repository/PostRepository.php

```diff
@@ -1107,11 +1108,29 @@ public function createMany(Collection $collection) : array
         $newPostIds = range($insertId, $insertId + $created - 1);
 
         // Loop over entities, and aggregate values by attribute
-        $postValues = collect($newPostIds)->combine($collection)->map(function ($entity, $id) {
-            if ($entity->values) {
-                $this->updatePostValues($id, $entity->values);
+        $postsById = collect($newPostIds)->combine($collection);
+
```

Comment:
If this doesn't make sense, I can try to make it more readable. It's an ugly transformation that *might* be clearer with a `foreach` instead of all the map/flatten/groupby stuff

Reply:
I'm ok with this style but can you expand the comment to be more like:
"reformat set of Posts as...group by 'key' bulk insert attribitues then return X..."